spec/integration/aws_tagged_items_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+describe "AWS Tagged Items" do
+  extend AWSSupport
+
+  when_the_chef_12_server "exists", organization: 'foo', server_scope: :context do
+    with_aws "when connected to AWS" do
+      it "aws_ebs_volume 'test_volume' created with default Name tag" do
+        expect_recipe {
+          aws_ebs_volume "test_volume"
+        }.to have_aws_ebs_volume_tags('test_volume',
+                       { 'Name' => 'test_volume' } )
+      end
+
+      it "aws_ebs_volume 'test_volume' tags are updated" do
+        expect_recipe {
+          aws_ebs_volume "test_volume_a" do
+            aws_tags :byebye => 'true'
+          end
+        }.to have_aws_ebs_volume_tags('test_volume_a',
+                                      { 'Name' => 'test_volume_a',
+                                        'byebye' => 'true'
+                                      })
+
+        expect_recipe {
+          aws_ebs_volume "test_volume_a" do
+            aws_tags :Name => 'test_volume_b', :project => 'X'
+          end
+        }.to have_aws_ebs_volume_tags('test_volume_a',
+                                      { 'Name' => 'test_volume_b',
+                                        'project' => 'X'
+                                      })
+      end
+
+      it "aws_ebs_volume 'test_volume' created with new Name tag" do
+        expect_recipe {
+          aws_ebs_volume "test_volume_2" do
+            aws_tags :Name => 'test_volume_new'
```

Comment:
Can you make the key here `'Name'` instead of `:Name` so we can test that works?